packages/xai/src/xai-chat-settings.ts

```diff
@@ -3,13 +3,28 @@ import { OpenAICompatibleChatSettings } from '@ai-sdk/openai-compatible';
 // https://console.x.ai and see "View models"
 export type XaiChatModelId =
   | 'grok-3-beta'
+  | 'grok-3'
+  | 'grok-3-latest'
   | 'grok-3-fast-beta'
+  | 'grok-3-fast'
+  | 'grok-3-fast-latest'
   | 'grok-3-mini-beta'
+  | 'grok-3-mini'
+  | 'grok-3-mini-latest'
   | 'grok-3-mini-fast-beta'
-  | 'grok-2-1212'
+  | 'grok-3-mini-fast'
+  | 'grok-3-mini-fast-latest'
   | 'grok-2-vision-1212'
-  | 'grok-beta'
+  | 'grok-2-vision'
+  | 'grok-2-vision-latest'
+  | 'grok-2-image-1212'
+  | 'grok-2-image'
+  | 'grok-2-image-latest'
+  | 'grok-2-1212'
+  | 'grok-2'
+  | 'grok-2-latest'
   | 'grok-vision-beta'
+  | 'grok-beta'
   | (string & {});
```

Comment:
maybe take out the `beta` ones

Reply:
i think now we've shipped it, it would be a breaking change - so we should probably keep it

Reply:
i see you added a comment here, but `grok-beta` has no other aliases. I can remove the `grok-3-beta` ones though cause they have aliases

Reply:
@lgrammel updated it if that's what you meant